Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -153,8 +153,7 @@ extension NavigationMapView {
             }
         }
         
-        func onViewDidAppear() {
-            showRouteIfNeeded()
```

Comment:
Previous delegate method name was incorrect, so it was never called, not sure if we really need `navigationViewDidAppear(_:)` anyway. Call to `showRouteIfNeeded()` is not needed either as it'll be called each time `MapView` style changes.